.github/workflows/osgeo4w.yml

```
@@ -51,31 +51,26 @@ jobs:
           package-dir: "D:/OSGeo4W_pkg"
           packages: |
             cairo-devel
-            fftw
             freetype-devel
             gdal-devel
-            gdal-ecw
-            gdal-mrsid
             geos-devel
+            libjpeg-turbo-devel
             liblas-devel
             libpng-devel
             libpq-devel
             libtiff-devel
-            libxdr
             netcdf-devel
-            pdal-devel
-            pdcurses
             proj-devel
             python3-core
             python3-jupyter
-            python3-matplotlib
             python3-numpy
```

Since we have some tests in the grass.benchmark (python/grass/benchmark/testsuite/)using matplotlib, should we be keeping it installed?
```suggestion
            python3-matplotlib
            python3-numpy
```

---

Yes, matplotlib should be there